opensuse_buildservice/notqmail.dsc

```diff
@@ -0,0 +1,13 @@
+Format: 1.0
+Source: notqmail
+Section: Misc
+Priority: extra
+Binary: notqmail
+Architecture: any
+Version: 1.08-1.1
+Maintainer: Manvendra Bhangui <manvendra@indimail.org>
+Homepage: https://github.com/notqmail/notqmail
+Debtransform-Files-Tar: debian.tar.gz
+Debtransform-Tar: notqmail-1.08.tar.xz
+Standards-Version: 3.9.1
+Build-Depends: cdbs, debhelper (>= 7.0.50~), gcc, g++, automake, bison (>= 1.2), mime-support, m4, gawk
```

Comment:
Why do we need g++, automake, bison, m4, and gawk?

Reply:
Don't think they are needed here as we don't have any calls to awk, yacc, etc in our Makefile. g++, automake also can be removed as I have copy pasted from my project which uses ALT-N's libdkim for DKIM.